### ISSUE TITLE
.exe file missing

### ISSUE THREAD

**Issue by kwalkington** [sofmeright/VirtualDisplayDriver_Wizard#5]
Is the .exe file missing from the .zip folder?

**Comment by SoFMeRight** [comment]
Shouldn't be. I did a weird release structure didn't quite understand GitHub at first but everything should be available.
The latest release is [here](https://github.com/sofmeright/VirtualDisplayDriver_Wizard/releases/download/beta_v0.614/PRPlanIT.com-VirtualDisplayDrv_Wiz.zip)

**Comment by kwalkington** [comment]
You're right.  The green code button links to a different package download then the latest release you just provided.  How would I get to the link you provided?

**Comment by kwalkington** [comment]
In your picture of the integration with sunshine, you have 4 lines.
Is the first the ..._Wiz.exe
Is the second the ..._Wiz.ahk?
I'm assuming the third line is NOT needed and was for your use case?
The fourth points to Qres, but what is the rest of that line?
The picture does not show: "cmd /C SunshineIntegration.bat %SUNSHINE_CLIENT_WIDTH% %SUNSHINE_CLIENT_HEIGHT% %SUNSHINE_CLIENT_FPS%".  So where do we run that?

**Comment by SoFMeRight** [comment]
> You're right. The green code button links to a different package download then the latest release you just provided. How would I get to the link you provided?

If you go to the releases segment on the right side of the main page it should be in a zip folder there.

> Is the first the ..._Wiz.exe & Is the second the ..._Wiz.ahk?

If you are using sunshine you will want to use the command you gave as also below in any of the spaces. I never updated the example photo. And also check the box for it to run elevated.
"cmd /C SunshineIntegration.bat %SUNSHINE_CLIENT_WIDTH% %SUNSHINE_CLIENT_HEIGHT% %SUNSHINE_CLIENT_FPS%"
The #1 and #2 in the example picture were actually attempts at piping the commands thru Sunshine via the Wizard's ".exe", in my tests it does not work unless I use the sunshine integration ".bat".
#3 is launching VirtualHere which is a USB sharing application I use to bind USB devices physically via the client to the Server.

> The fourth points to Qres, but what is the rest of that line?

The rest of the line is:
"cmd /C C:\_Staging\_Utilities\QRes.exe /X:%SUNSHINE_CLIENT_WIDTH% /Y:%SUNSHINE_CLIENT_HEIGHT% /R:%SUNSHINE_CLIENT_FPS%"

If you want to see a little bit of how I use this I do have a work in progress write-up of my setup and linked into the Readme.
https://kb.precisionplanit.com/books/windows-1011/page/transform-your-pc-into-a-dedicated-gaming-machine-headless-gaming-rig-tutorial

Sorry for the delayed response! Hope you got it sorted, otherwise hope that this helps!
